docs/model_developers/configurationmodel.rst

```diff
@@ -12,73 +13,62 @@ contains the source code of the configuration model. This project also matches w
 :term:`project` defined on the server, from which multiple :term:`environments<environment>` can be
 deployed.
 
-.. note::
-
-    Inmanta requires that a project is a git repository. This is not strictly required when using
-    the embedded or push to server model (see :doc:`/architecture`). However in the autonomous
-    server model, this is the only method to get the configuration code on the server. Additionally,
-    it is also a good practive to version control your infrastructure code.
-
-    Typically branches in this git repository are used to define multiple environments (if the
-    differ in code)
-
 .. code-block:: sh
-    :linenos:
+  :linenos:
 
-    mkdir hello-world
-    cd hello-world
-    git init
+  pip install cookiecutter
+  cookiecutter gh:inmanta/inmanta-project-template
 
 Inside the project the compiler expects a ``project.yml`` file that defines metadata about the project,
 the location to store modules, repositories where to find modules and possibly specific versions of
 modules. project.yml defines the following settings:
 
-    * ``name`` An optional name of the project.
+    * ``name`` The name of the project.
     * ``description`` An optional description of the project
-    * ``modulepath`` This value is a list of paths where Inmanta should search for modules. Paths
-      are separated with ``:``
-    * ``downloadpath`` This value determines the path where Inmanta should download modules from
-      repositories. This path is not automatically included in in modulepath!
+    * ``author``  The author of the module
+    * ``author_email`` The contact email address of author
+    * ``license`` License the module is released under
+    * ``copyright`` Copyright holder name and date.
     * ``install_mode`` This key determines what version of a module should be selected when a module
       is downloaded. This is used when the module version is not "pinned" in the ``requires`` list.
       The available values are:
 
         * release (default): Only use a released version, that is compatible with the current
-          compiler. A version is released when there is a tag on a commit. This tag should be a
```

Comment:
Is this explained somewhere else in the documentation?

Reply:
yes, linked below: :ref:`project_yml`